### DISCUSSION TITLE
More than a month and haven't any received any response fromt the Github Support Team

### DISCUSSION THREAD

**Discussion by bhavya3024-temp** [community/community#136819] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### GitHub Feature Area

Other

### Body

Hi there,
I have already created my created more than a month ago: https://support.github.com/ticket/personal/0/2900494
But there is no response from the support team from a month.
It was related to my suspended github account having username: bhavya3024.
For Official Github members, can you please flag my ticket and resolve it as soon as possible?
I want to add more  thing, some scammers are taking the advantage of the situation through linkedin as well. (so called cybersecurity experts).
Thanks!

**Comment by ghostinhershell** [comment]
Hi @bhavya3024-temp,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
